game-core/src/main/java/games/strategy/engine/data/DefaultUnitFactory.java

```diff
@@ -1,5 +1,8 @@
 package games.strategy.engine.data;
 
+/**
+ * Default implementation of {@link IUnitFactory}.
```

Comment:
"Re-stating the code" comment smell here for me, `DefaultUnitFactory implements IUnitFactory` has a lot of overlap with ` Default implementation of {@link IUnitFactory}.`

To help this, I would describe what is involved with a default implementation, how/what made this to be a concrete instance and what kinds of things are backing the implementation.

Reply:
`DefaultUnitFactory` is only used by test code; all production code uses an anonymous `IUnitFactory` implementation that always returns units of type `TripleAUnit`.

I will inline this class into the applicable test code in a follow-up PR.